xpk.py

```diff
@@ -1160,6 +1289,14 @@ def run_gke_cluster_create_command(args) -> int:
       ' --scopes=storage-full,gke-default'
       f' {args.custom_cluster_arguments}'
   )
+  if _USE_SERVICE_ACCOUNT:
+    service_account_name = get_service_account_name(args)
+    service_account_exists = check_if_service_account_exists(args)
+    if service_account_exists:
+      command += f' --service-account={service_account_name}'
+    else:
+      xpk_print(f'Service Account: {service_account_name} does not exist in the project.'
```

Comment:
nit comment here saying: we will not create the service account for previously created GKE clusters. I think this will be useful to reminder us why we have this case.